### ISSUE TITLE
Simply doesn't open

### ISSUE THREAD

**Issue by Chancebest123** [alievk/avatarify-desktop#25]
When opening the EXE file, it simply does nothing.
I have a Ryzen 7 2700x and RTX 2070
I have open with admin as well as checked task manager and nothing shows forit, it just simply does not do anything, no error message, no nothing, kinda like a dud. Any ideas?

**Comment by Feelosov** [comment]
Yep, confirm. The computer thinks, thinks, thinks aaand ... nothing.

**Comment by JohanAR** [comment]
> Yep, confirm. The computer thinks, thinks, thinks aaand ... nothing.

You mean you have exactly the same hardware config as the person who opened this issue?

**Comment by Feelosov** [comment]
No, I just didn't want to breed unnecessary topics.

**Comment by JohanAR** [comment]
@Feelosov if you have a RTX30xx graphics card, then it's currently not supported. See issue https://github.com/alievk/avatarify-desktop/issues/9

If you have a different GPU, please post your hardware and OS information here. So far I've only seen a single report, i.e. this issue, about the app not starting on a RTX20xx card

**Comment by Feelosov** [comment]
Ok, thx. Windows 10, GTX660, 16 RAM, hard 1Tb (free). If I can help with other technical information, let me know.

**Comment by JohanAR** [comment]
> Ok, thx. Windows 10, GTX660, 16 RAM, hard 1Tb (free). If I can help with other technical information, let me know.

Sorry, I think your GPU is too old to run any of the latest versions of CUDA, which Avatarify Desktop depends on. I'm afraid even if it did start, it would lack the computational power to achieve any usable frame rate. GTX 1070 or better is recommended.

You could check out the [mobile version of Avatarify](https://avatarify.ai) if you want

**Comment by Feelosov** [comment]
It's OK. I didn't really hope. Suddenly someone with the same video card will look in. Thanks for the help.

**Comment by dfyx** [comment]
Similar problem with Windows 10 Pro, AMD Ryzen 5 3600X, RTX 3070. I had the application open once after install but couldn't get an image. Closed it, tried to reopen it, nothing. CPU for the process is at 10%, RAM fluctuates between 200 MB and 1 GB but no window opens.

**Comment by JohanAR** [comment]
@dfyx it currently doesn't work on RTX 30xx cards, see #9

**Comment by FPcrab** [comment]
E5 2678v3 32GB RAM, RTX 2060
app runs, make calibration, when i click to image below - app crash witn no error.

**Comment by Cherry21vn** [comment]
i had the same problem when i installed it then opened it and waited for a very long time without any windows opened my config CPU 10100f GPU gt 710 sl ,8gb ram let me know ?

**Comment by JohanAR** [comment]
@Cherry21vn I don't think that graphics card is powerful enough to run Torch, which is the library that Avatarify uses to do all the GPU computing

**Comment by gschian0** [comment]
it doesn't run on a 3070 laptop gpu?

**Comment by JohanAR** [comment]
@gschian0 I don't think it works on any 30xx cards, but if someone updates dependencies so that it gets Torch + CUDA (I think this is the problem) which works with those GPUs, then I guess a 3070 mobile version ought to be powerful enough to run it

**Comment by gschian0** [comment]
Cool!

**Comment by badmuspro** [comment]
I have the same issue please can you documents step by step tutorial or a video and drop the link to learn and resolve it.. I downloaded the avatarify desktop version and I open but nothing happened I keep waiting it doesn’t open.. then I use AMD app and much from inside it open but if I click on any image then it stucks.. my system is powerful Asus Rog Zephyrus G14
AMD Ryzen 9 5900H
Nvidia RTX 3060
16 Gb ram
1tb ssd.
I think this should run it because it’s more powerful than GTX 1070 recommended.. and it’s mention in GitHub to run on Nvidia 1070 or better ( I.e from 1070 upward )

**Comment by JohanAR** [comment]
@badmuspro it's not that simple, see #9
